### DISCUSSION TITLE
UERANSIM not generating GTP packets, it sends plain TCP packets

### DISCUSSION THREAD

**Discussion by Lark2674** [aligungr/UERANSIM#339] [Q&A]
Hi All,

We are 5G core dev Team from ACL Digital, we are using UERANSIM to test our 5G Core code.
Able to establish PDU session between UERANSIM(UE and GNB in one vm) and 5G Core in another VM.

While sending traffic using ./nr-binder 192.168.128.12 curl google.com command we see plain TCP packets instead of GTP packets.

We are observing GTP error in GNB logs. Looks like UE context becoming null.
[2021-06-02 10:17:36.906] [ngap] [info] PDU session resource(s) setup for UE[1] count[1]
[2021-06-02 10:17:36.907] [gtp] [error] PDU session resource could not be created, UE context with ID[1] not found
[2021-06-02 10:17:51.807] [gtp] [error] Uplink data failure, PDU session not found. UE[1] PSI[1]

Attaching the mail discussion we initiated with all the logs and config we are using.
Please suggest a way forward for us to resolve the issue.

Thanks,
Lark Kannan

[ueransim.zip](https://github.com/aligungr/UERANSIM/files/6589748/ueransim.zip)

**Comment by aligungr** [comment]
Hi @Lark2674 

~Please attach a PCAP file in order to analyze the problem~

**PS.** I obtained a PCAP file from one of your colleagues, and replied with an email.

Thanks

**Reply by Lark2674** [reply]
Hi @aligungr,

Thanks for the reply.

Regards,
Lark Kannan
